gulpfile.js

```diff
@@ -58,18 +58,18 @@ function lesslint() {
 }
 
 function less() {
-	return gulp.src('src/**/*.less')
+	return gulp.src('src/sass/**/*.scss')
```

Comment:
For sass it makes sense, but for ts, we should only have it into /src/app/ right ?